### DISCUSSION TITLE
Ability to watch multiple replays side by side

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#18639] [Ideas]
Being able to watch multiple players in a multiplayer lobby at the same time is a really cool feature, and there is a demographic of people who enjoy [comparing replays in real time](https://www.youtube.com/c/001thefish), so I think it would be very nice to be able to view multiple replays side by side and compare their stats (like pp and score) in real time in the client.

**Comment by PercyDan54** [comment]
Fwiw I had a branch already with the similar feature, by messing around with the multiplayer spectator screen.
It's not PRable since it's low code quality and there's some hacks, but basically usable I guess.
I'd also like to see this feature added

![image](https://user-images.githubusercontent.com/50285552/173173090-c4ed8e26-db39-4273-af10-b70fed1d4260.png)

**Comment by the-wildguy** [comment]
or maybe having the ability to have them overlaid with different cursor colours
